eventarc/pubsub/README.md

```diff
@@ -0,0 +1,45 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes Pub/Sub events using 
+[the CloudEvents SDK](https://github.com/cloudevents/sdk-python).
+
+## Quickstart
+
+Deploy your Cloud Run service:
+
+```sh
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/eventarc-pubsub
+gcloud run deploy eventarc-pubsub \
+ --image gcr.io/$(gcloud config get-value project)/eventarc-pubsub
```

Comment:
Should you add `--platform managed`?

Reply:
Was thinking of not including it, but since we have a separate Anthos README, we should probably include it.